bin/isolated-functions.sh

```diff
@@ -340,9 +340,15 @@ ebegin() {
 	LAST_E_LEN=$(( 3 + ${#RC_INDENTATION} + ${#msg} ))
 	LAST_E_CMD="ebegin"
 	if [[ -v EBEGIN_EEND ]] ; then
-		eqawarn "QA Notice: ebegin called, but missing call to eend (phase: ${EBUILD_PHASE})"
+		# Already a call to ebegin
+		local prev="${EBEGIN_EEND[-1]}"
+		if [[ "${prev}" == "${FUNCNAME[1]}" ]] ; then
+			eqawarn "QA Notice: ebegin called in ${prev}, but missing call to eend (${FUNCNAME[1]})"
```

Comment:
err, maybe not the best message, as ${prev} and ${FUNCNAME[1]} will be the same string :-)

How about:
```
QA Notice: ebegin already called in ${prev}, but missing call to eend
```
?